### PR TITLE
[turtlebot_description] adds orbbec astra urdfs and mesh

### PR DESCRIPTION
**Pull request by bit-pirate**
To verify this use the model viewer:

First, set your turtlebot workspace environment variables, then

```
$ export TURTLEBOT_3D_SENSOR=astra
$ roslaunch turtlebot_rviz_launchers view_model.launch
```

### CODE REVIEW

File: turtlebot_description/robots/kobuki_hexagons_astra.urdf.xacro [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<!--
+    - Base      : kobuki
+    - Stacks    : hexagons
+    - 3d Sensor : kinect
```
Comment (tfoote) [inline turtlebot_description/robots/kobuki_hexagons_astra.urdf.xacro:5]: astra here

**Comment by tfoote** [comment]
Merged on the commandline with the above comment fixed up.  4ad9522

**Comment by bit-pirate** [comment]
Thanks, Tully. The camera internal links would need to be verified. I don't have the measurements at hand.

https://github.com/turtlebot/turtlebot/issues/221

**Comment by stonier** [comment]
Thanks @tfoote...got to it before I could.

**Pull request closed without merging**